manifests/base/api-deployment.yaml

```diff
@@ -105,6 +105,8 @@ spec:
               value: https://$(BASE_DOMAIN)
             - name: ADMIN_CLIENT_SECRET
               value: '$(ADMIN_CLIENT_SECRET)'
+            - name: API_HOST_NAME
+              value: 'api.notification-canada-ca.svc.cluster.local:6011'
```

Comment:
We don't want to get these from a variable as we do most of the others? just curious.

Reply:
They are static based on the way the k8s cluster is configured, the others are variable based on how you want to configure it